jenkins/pipeline.groovy

```diff
@@ -234,8 +248,11 @@ pipeline {
                 timeout(time: pipeline_timeout, unit: 'HOURS')  {
                     retry(3) {
                         git branch: '8.0', url: 'https://github.com/Percona-Lab/ps-build'
-                        withCredentials([string(credentialsId: 'MTR_VAULT_TOKEN', variable: 'MTR_VAULT_TOKEN')]) {
-                            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'c8b933cd-b8ca-41d5-b639-33fe763d3f68', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                        withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'c8b933cd-b8ca-41d5-b639-33fe763d3f68', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                            withCredentials([
+                                string(credentialsId: 'MTR_VAULT_TOKEN', variable: 'MTR_VAULT_TOKEN'),
```

Comment:
`MTR_VAULT_TOKEN` is probably not used any more.

Reply:
I'll remove it